source/gloperate-qtquick/include/gloperate-qtquick/Application.h

```diff
@@ -0,0 +1,123 @@
+
+#pragma once
+
+
+#include <QGuiApplication>
+#include <QString>
+
+#include <gloperate/base/Environment.h>
+
+#include <gloperate-qt/base/UpdateManager.h>
+
+#include <gloperate-qtquick/QmlEngine.h>
+
+
+namespace gloperate_qtquick
+{
+
+
+/**
+*  @brief
+*    Qt application object that initializes gloperate automatically
+*
+*  @remarks
+*    At the beginning of your application, call Application::initialize().
```

Comment:
The initialization of the Application is currently missing in the `main.cpp`s but they are integrated in #322.